src/pages/taskHistory/BuildVariantSelector.tsx

```diff
@@ -48,35 +45,25 @@ const BuildVariantSelector: React.VFC<BuildVariantSelectorProps> = ({
 
   const { buildVariantsForTaskName } = data || {};
 
-  const handleSearch = (options: BuildVariantTuple[], match: string) =>
-    options.filter(
-      (option) =>
-        option.buildVariant.includes(match) ||
-        option.displayName.includes(match)
-    );
-
   return (
     <Container>
-      <SearchableDropdown
+      <Combobox
```

Comment:
I assume this is a bug with the component but it's still annoying 😢 
https://user-images.githubusercontent.com/9298431/217654957-e368ed91-9acd-4a16-95f7-fd5a045ba6b5.mov

Reply:
I'll raise this with the LG folks